treeherder/perf/migrations/0053_performancealertsummary_original_push.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.1.1 on 2024-10-31 14:38
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def copy_push_to_original_push(apps, schema_editor):
+    alerts = apps.get_model('perf', 'PerformanceAlertSummary')
+    for alert in alerts.objects.all():
+        alert.original_push = alert.push
+        alert.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "model",
+            "0032_rename_failureline_job_guid_repository_failure_lin_job_gui_b67c6d_idx_and_more",
+        ),
+        (
+            "perf",
+            "0052_rename_performancedatum_repository_signature_push_timestamp_performance_reposit_c9d328_idx_and_more",
+        ),
```

Comment:
```suggestion
        (
            "perf",
            "0052_rename_performancedatum_repository_signature_push_timestamp_performance_reposit_c9d328_idx_and_more",
        ),
```

Reply:
This is most likely a mistake, usually we only have one dependency, which is the previous migration.